### PR TITLE
fozzie-components@v3.45.0 - Ability to load individual Storybook stories via launch.json

### PR DESCRIPTION
**Pull request by siggerzz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,7 +82,11 @@ To run storybook first you must run the following at the the top level
 
 `yarn build` to build the require packages for storybook to run.
 
-Then run `yarn storybook:serve` to start storybook, which will open on `localhost:6006`.
+Then run `yarn storybook:serve` to start storybook, which will open on `localhost:8080`.
+
+
```
Comment (ashleynolan) [inline README.md:87]:
```suggestion

### Running a single component in Storybook

```

File: README.md [hunk 2]
```diff
@@ -82,7 +82,11 @@ To run storybook first you must run the following at the the top level
...
+If you wish to view a single component in Storybook during development, you can do this by opening the components `*.stories.js` file in your IDE and running the `Storybook - Run Currently Open Story File` task. (This only works with VS Code).
+This is particuarly useful as it prevents a bug where Storybook pulls in other components stylesheets ([See here](https://github.com/storybookjs/storybook/issues/729)).
```
Comment (ashleynolan) [inline README.md:92]:
```suggestion

This is particuarly useful as it prevents a bug where Storybook pulls in other components stylesheets ([See here](https://github.com/storybookjs/storybook/issues/729)).
```

File: packages/storybook/config/storybook/story.config.js [hunk 3]
```diff
@@ -22,6 +22,22 @@ const getChangedPackageStories = () => {
     return storyPaths;
 };
 
+const getStoryFiles = () => {
+
+    if(process.env.VS_DEBUGGER) {
```
Comment (ashleynolan) [inline packages/storybook/config/storybook/story.config.js]: Worth adding some comments to this function for those new to this conditional

File: packages/storybook/config/storybook/story.config.js [hunk 4]
```diff
@@ -22,6 +22,22 @@ const getChangedPackageStories = () => {
...
+    else if(process.env.CHANGED_ONLY) {
+        return getChangedPackageStories();
+    }
+    else {
+        return [
+            '../../../**/*.stories.@(js|mdx)',
+            '../../../../stories/**/*.stories.@(js|mdx)'
+        ];
+    }
+}
```
Comment (ashleynolan) [inline packages/storybook/config/storybook/story.config.js]:
```suggestion
    if (process.env.VS_DEBUGGER) {
        return [process.env.CURRENT_STORY_FILE];
    }
    else if (process.env.CHANGED_ONLY) {
        return getChangedPackageStories();
    }
    else {
        return [
            '../../../**/*.stories.@(js|mdx)',
            '../../../../stories/**/*.stories.@(js|mdx)'
        ];
    }
}
```

File: README.md [hunk 5]
```diff
@@ -82,7 +82,14 @@ To run storybook first you must run the following at the the top level
 
 `yarn build` to build the require packages for storybook to run.
 
-Then run `yarn storybook:serve` to start storybook, which will open on `localhost:6006`.
+Then run `yarn storybook:serve` to start storybook, which will open on `localhost:8080`.
+
+
+### Running a single component in Storybook
+
+If you wish to view a single component in Storybook during development, you can do this by opening the components `*.stories.js` file in your IDE and running the `Storybook - Run Currently Open Story File` task. (This only works with VS Code).
```
Comment (JoshuaNg2332) [inline README.md]:
```suggestion
If you wish to view a single component in Storybook during development, you can do this by opening the components `*.stories.js` file in your IDE and running the `Storybook - Run Currently Open Story File` task which can be found in the debug tab on the left. (This only works with VS Code).
```

**Review by JoshuaNg2332 — APPROVED** [review]

**Review by ashleynolan — APPROVED** [review]

**Pull request merged**